src/Oro/Bundle/PricingBundle/Model/PriceListTreeHandler.php

```diff
@@ -76,11 +76,15 @@ public function __construct(
      * @param Website|null $website
      * @return CombinedPriceList|null
      */
-    public function getPriceList(Customer $customer = null, Website $website = null)
+    public function getPriceList(Customer $customer = null, $website = null)
     {
         if (!$website) {
             $website = $this->websiteManager->getCurrentWebsite();
         }
+        if (!$website) {
```

Comment:
Website manager already return default website when it's needed, see https://github.com/oroinc/customer-portal/blob/3.0.0/src/Oro/Bundle/WebsiteBundle/Manager/WebsiteManager.php#L70